snuba/migrations/migration.py

```diff
@@ -101,3 +110,25 @@ def backwards(self, context: Context) -> None:
             op.execute(local=True)
         logger.info(f"Finished reversing: {migration_id}")
         update_status(Status.NOT_STARTED)
+
+    def __dry_run(
+        self,
+        local_operations: Sequence[Operation],
+        dist_operations: Sequence[Operation],
+    ) -> None:
+
+        for op in local_operations:
+            if isinstance(op, SqlOperation):
+                print(op.format_sql())
+            else:
+                print("Non SQL operation")
+
+        for op in dist_operations:
```

Comment:
Some way to separate the output of the local migrations from the distributed ones would make it easier to read.